### PR TITLE
Battle of Baltimore

### PR DESCRIPTION
**Pull request by csokolove**
climb broke lol moment

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -4,6 +4,9 @@
 
 package frc.robot;
 
+import org.ejml.data.ZMatrix;
```
Comment (csokolove) [inline src/main/java/frc/robot/Robot.java:7]: Delete this

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -73,11 +73,12 @@ public RobotContainer() {
         /* Start USB camera capture */
         CameraServer.startAutomaticCapture();
         /* Set default commands */
-        drivetrain.setDefaultCommand(curveDrive);
+        drivetrain.setDefaultCommand(arcadeDrive);
```
Comment (csokolove) [inline src/main/java/frc/robot/RobotContainer.java:76]: swap these

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -73,11 +73,12 @@ public RobotContainer() {
...
         // drivetrain.setDefaultCommand(arcadeDrive);
         turret.setDefaultCommand(lockTurret);
         /* Configure the button bindings */
         configureButtonBindings();
+        /* Turn on Limelight LEDs */
```
Comment (csokolove) [inline src/main/java/frc/robot/RobotContainer.java:81]: remove this line

File: src/main/java/frc/robot/commands/SpinUpShooter.java [hunk 4]
```diff
@@ -15,8 +15,8 @@ public class SpinUpShooter extends CommandBase {
     private final Limelight limelight;
```
Comment (csokolove) [inline src/main/java/frc/robot/commands/SpinUpShooter.java:15]: undo this

File: src/main/java/frc/robot/subsystems/Shooter.java [hunk 5]
```diff
@@ -144,12 +144,13 @@ public double timeOfFlight(double distance) {
      * @return  a calculated velocity from shooterRegression if the target is
```
Comment (csokolove) [inline src/main/java/frc/robot/subsystems/Shooter.java:144]: undo this

**Review by csokolove — COMMENTED** [review]
Changes fixed

**Comment by csokolove** [comment]
@lacedwithennui Can you approve these?

**Comment by lacedwithennui** [comment]
gotchu fam

File: src/main/java/frc/robot/Robot.java [hunk 6]
```diff
@@ -28,6 +30,7 @@ public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
     // autonomous chooser on the dashboard.
     m_robotContainer = new RobotContainer();
+    m_robotContainer.limelight.setLEDs(LedValues.ON);
```
Comment (lacedwithennui) [inline src/main/java/frc/robot/Robot.java:33]: remove all of the limelight stuff from Robot.java

File: src/main/java/frc/robot/RobotContainer.java [hunk 7]
```diff
@@ -38,14 +38,14 @@ public class RobotContainer {
     private final Kicker rightKicker = Kicker.createRightKicker();
     private final Kicker leftKicker = Kicker.createLeftKicker();
     protected final Turret turret = new Turret();
-    private final Limelight limelight = new Limelight(drivetrain, turret);
```
Comment (lacedwithennui) [inline src/main/java/frc/robot/RobotContainer.java:41]: revert this to private after removing the limelight stuff from Robot.java

**Review by lacedwithennui — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Robot.java (hunk 6 above)
Comment (csokolove) [reply src/main/java/frc/robot/Robot.java:33]: didn't we decide this was the way to fix it? or was it a sketchy way

**Review by lacedwithennui — APPROVED** [review]

**Pull request merged**